executor/adapter.go

```diff
@@ -843,6 +843,7 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool, hasMoreResults bool) {
 		PlanDigest:     planDigest,
 		Prepared:       a.isPreparedStmt,
 		HasMoreResults: hasMoreResults,
+		PlanFromCache:  sessVars.FoundInPlanCache,
```

Comment:
`FoundInPlanCache` means "if the current executing statement is from plan cache", while `PrevFoundInPlanCache` means "if the last executing statement is from plan cache". Currently when we execute 'select last_plan_from_cache', the value of `PrevFoundInPlanCache` is shown as result. So I wonder which value is needed here.

Reply:
`LogSlowQuery` is called before finishing execution of current query, so we should use `FoundInPlanCache` I think.